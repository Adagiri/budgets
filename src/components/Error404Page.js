import React from "react";
import { Link } from "react-router-dom";

const Error404Page = props => (
  <div>
    <h1>
      404! <Link to="/">Go home</Link>
    </h1>
  </div>
);

export default Error404Page;
