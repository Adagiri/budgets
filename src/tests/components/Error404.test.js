import React from 'react'
import Error404Page from '../../components/Error404Page';
import { shallow } from 'enzyme';

test('should render error page correctly', () => {
    const wrapper = shallow( <Error404Page />);
    expect(wrapper).toMatchSnapshot();
});