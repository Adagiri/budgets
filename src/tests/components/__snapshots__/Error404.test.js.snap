// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render error page correctly 1`] = `
<div>
  <h1>
    404! 
    <Link
      to="/"
    >
      Go home
    </Link>
  </h1>
</div>
`;
